docs/modules/publisher-dmmigration.md

```diff
@@ -1,6 +1,16 @@
 [Home](../../readme.md)
 
-# Migration path for IoT Hub Direct Methods
+# Migration path for publisher version 2.5.x to 2.8.2 - configuration file structure (pn.json)
+**TODO**
+2.8.2 must work in standalone mode for backwards compatibility with 2.5.x functionality
+
+# Migration path for publisher version 2.5.x to 2.8.2 - CLI arguments
```

Comment:
Documents should usually contain only one top level heading. Let's keep the original one (`# Migration path for IoT Hub Direct Methods`) or something like that, maybe more generic. And all of those should become 2nd level heading (##).

Reply:
got it!

Reply:
I agree. There should be only one top level heading.